elements/howto-checkbox/README.md

```diff
@@ -1,5 +1,180 @@
-A `HowToCheckbox` represents a boolean option in a form. The most common type
+# `<howto-checkbox>`
```

Comment:
Remove this L1 headline, it’s already generated by the [DevSite template](https://github.com/GoogleChrome/howto-components/blob/master/site-resources/element.tpl.md#howto-components--ittitle--page-title-) because it needs some special markup.